README.rst

```diff
@@ -1,3784 +1,221 @@
-in the Hazelcast Management Center Reference Manual for more information
-on the client statistics.
+You can also use our ZIP or TAR
+`distributions <https://hazelcast.org/imdg/download/archives/#hazelcast-imdg>`__.
+Once you have downloaded, you can start the Hazelcast member using
+the ``bin/start.sh`` script.
 
-    NOTE: Statistics sent by Hazelcast Python client 4.0 are compatible
-    with Management Center 4.0. Management Center 4.2020.08 and newer
-    versions will be supported in version 4.1 of the client.
+Client
+~~~~~~
 
-7.9.2 Logging Configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code:: bash
 
-Hazelcast Python client uses Python’s builtin ``logging`` package to
-perform logging.
+   pip install hazelcast-python-client
 
-All the loggers used throughout the client are identified by their
-module names. Hence, one may configure the ``hazelcast`` parent logger
-and use the same configuration for the child loggers such as
-``hazelcast.lifecycle`` without an extra effort.
+Overview
+--------
 
-Below is an example of the logging configuration with ``INFO`` log level
-and a ``StreamHandler`` with a custom format, and its output.
+Usage
+~~~~~
 
 .. code:: python
 
-    import logging
     import hazelcast
 
-    logger = logging.getLogger("hazelcast")
-    logger.setLevel(logging.INFO)
-
-    handler = logging.StreamHandler()
-    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-    handler.setFormatter(formatter)
-    logger.addHandler(handler)
-
+    # Connect to Hazelcast cluster.
     client = hazelcast.HazelcastClient()
 
-    client.shutdown()
-
-**Output**
+    # Get or create the "distributed-map" on the cluster.
+    distributed_map = client.get_map("distributed-map")
 
-::
+    # Put "key", "value" pair into the "distributed-map" and wait for
+    # the request to complete.
+    distributed_map.set("key", "value").result()
 
-    2020-10-16 13:31:35,605 - hazelcast.lifecycle - INFO - HazelcastClient 4.0.0 is STARTING
-    2020-10-16 13:31:35,605 - hazelcast.lifecycle - INFO - HazelcastClient 4.0.0 is STARTED
-    2020-10-16 13:31:35,605 - hazelcast.connection - INFO - Trying to connect to Address(host=127.0.0.1, port=5701)
-    2020-10-16 13:31:35,622 - hazelcast.lifecycle - INFO - HazelcastClient 4.0.0 is CONNECTED
-    2020-10-16 13:31:35,622 - hazelcast.connection - INFO - Authenticated with server Address(host=172.17.0.2, port=5701):7682c357-3bec-4841-b330-6f9ae0c08253, server version: 4.0, local address: Address(host=127.0.0.1, port=56752)
-    2020-10-16 13:31:35,623 - hazelcast.cluster - INFO -
+    # Try to get the value associated with the given key from the cluster
+    # and attach a callback to be executed once the response for the
+    # get request is received. Note that, the set request above was
+    # blocking since it calls ".result()" on the returned Future, whereas
+    # the get request below is non-blocking.
 
```

Comment:
nit: probably, a redundant line here.

Reply:
I don't like this extra space too but PEP8 recommends it 
https://www.python.org/dev/peps/pep-0008/#blank-lines 
`Surround top-level function and class definitions with two blank lines.`

Reply:
Didn't know that. Let's leave it as is then.